### PR TITLE
add support to load files from the directories in directories.json

### LINKED ISSUE #4: Add support for multiple maps directories in hgl-editor
Right now the hgl-editor only support loading and writing maps into the hac-maps/ directory.
It would be nice to support more locations.
This improvement require some changes in the way maps/boards are loaded and particularly to keep the directory they have been loaded from.

### PR DESCRIPTION
**Pull request by kaozdl**
This PR adds a directories.json attempts to fix #4 
you can add multiple directories there and load files from there.
Once the file is opened it can be saved to that place.
What's missing is to save to a directory when creating a new board.

### CODE REVIEW

File: directories.json [hunk 1]
```diff
@@ -0,0 +1 @@
+["hac-maps/"]
```
Comment (arnauddupuis) [inline directories.json]: Now that I see it, we should probably do just:
["hac-maps"]
And leave the os module concatenate the directory separators.

**Comment by arnauddupuis** [comment]
Really cool PR. Thanks a lot!
Do you want me to hold the merge for the "saving" part or do I open a separate issue?

File: directories.json (hunk 1 above)
Comment (kaozdl) [reply directories.json]: yup, that is valid too.
if we do that we have to change the code inside. these are very small changes

**Comment by kaozdl** [comment]
i can issue another one tomorrow for the saving part. as it is now it can save once it has oppened the file successfully.
But creating a new file it's a whole new thing.
Maybe we can create wizard for selecting the current folder in a similar fashion to the current_file and save there

File: directories.json (hunk 1 above)
Comment (kaozdl) [reply directories.json]: done :)

Comment (arnauddupuis) [reply directories.json]: Thanks man, you're awesome!

**Comment by arnauddupuis** [comment]
> i can issue another one tomorrow for the saving part. as it is now it can save once it has oppened the file successfully.
> But creating a new file it's a whole new thing.
> Maybe we can create wizard for selecting the current folder in a similar fashion to the current_file and save there

That's an excellent idea. I'll add the issue and merge that PR.
Again: thank you for your help.

**Pull request merged**